smtp.c

```diff
@@ -290,6 +436,63 @@ ratelimitspp_print(const unsigned long time) {
 	nd_set("error", aggregated_ratelimtspp.error);
 	nd_set("ratelimited", aggregated_ratelimtspp.ratelimited);
 	nd_end();
+	return fflush(stdout);
+}
+
+static
+void
+print_limits(struct vector * limit, const char * limit_name, const unsigned long time) {
+	struct limit_t * l;
+
+	struct vector newdim;
+	vector_init(&newdim, sizeof(struct limit_t));
+
+	for (int i = 0; i < limit->len; i++) {
+                l = vector_item(limit, i);
+                if (l->new) {
+                        vector_add(&newdim, l);
+		}
```

Comment:
mixed space-tab indentation. Could you please unify them to tabs only at the beginning of the line?